### ISSUE TITLE
Problems when I am trying to connect with Firebase Database (Instant app)

### ISSUE THREAD

**Issue by moisesvs** [willowtreeapps/android-instant-apps-demo#1]
Hi, 

I am haveing a problem when trying to connect the instant app with Firebase database. With the normal App there are not any problem only I have the problem with the Instant App.

I believe that the problem could be because Instant app is not signatured with nothing certificate for this reason it don't connect with Firebase's Database. How Could I do for this works?
Regards

**Comment by sunian** [comment]
did you copy your `google-services.json` into the base feature module? It's the `instantapps-demo-base` folder.

**Comment by moisesvs** [comment]
Yes. I have it into in instantapps-demo-base I followed your steps of the installation. Any idea more?.

Thanks a lot!

**Comment by sunian** [comment]
can you give more detail on what doesn't work exactly?
are you getting an error or stacktrace?
what gradle commands are you using?
what emulator are you running on?
note: Instant Apps currently don't work when deploying over adb to a physical device.

**Comment by iamkaan** [comment]
not sure if it's the same issue but I posted a questions on SO: https://stackoverflow.com/questions/48292901/firebase-support-library-dependency-conflict-for-instant-apps. any ideas?
